microsoft/PowerToys
# [KeyboardManager] Add icon

## Summary of the Pull Request
Added Segoe  MDL2 KeyboardClassic as KeyboardManager icon.

 
## References


## PR Checklist
* [x] Applies to #2869
* [] Tests added/passed
* [] Requires documentation to be updated
* [] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx


## Detailed Description of the Pull Request / Additional comments
![image](https://user-images.githubusercontent.com/9252240/81625024-9ca5f000-93ac-11ea-9df4-e11fb664766b.png)



## Validation Steps Performed
1. Launch PowerToys.
2. Open Settings
3. Go to KeyboardManager settings
4. Add a single key remap.
5. The Window's title bar icon should be the new one, the icon in the task bar should be the new one.
6. Add a shortcut.
7. The Window's title bar icon should be the new one, the icon in the task bar should be the new one.
